ManusGlove/src/ManusGloveHelper.cpp

```diff
@@ -896,6 +1017,86 @@ std::string ManusGloveHelper::ConvertDeviceFamilyTypeToString(DeviceFamilyType p
     }
 }
 
+std::string manusGlove::ManusGloveHelper::ConvertChainTypeToString(ChainType p_Type)
+{
+    switch (p_Type)
+    {
+    case ChainType_FingerThumb:
+        return "Thumb";
+    case ChainType_FingerIndex:
+        return "Index";
+    case ChainType_FingerMiddle:
+        return "Middle";
+    case ChainType_FingerRing:
+        return "Ring";
+    case ChainType_FingerPinky:
+        return "Pinky";
+    case ChainType_Head:
+        return "Head";
+    case ChainType_Neck:
+        return "Neck";
+    case ChainType_Spine:
+        return "Spine";
+    case ChainType_Shoulder:
+        return "Shoulder";
+    case ChainType_Arm:
+        return "Arm";
+    case ChainType_Hand:
+        return "Hand";
+    case ChainType_Foot:
+        return "Foot";
+    case ChainType_Toe:
+        return "Toe";
+    case ChainType_Invalid:
+        return "Invalid";
+    case ChainType_Leg:
+        return "Leg";
+    default:
+        break;
+    }
+    return "Unknown";
+}
+
+inline std::string manusGlove::ManusGloveHelper::ConvertSideToString(Side p_Side)
```

Comment:
just a cuorisity, why inlining only for this conversion method, a particular reason?

Reply:
No real reason. I did something similar to some existing method there were above.